src/main/java/seedu/address/logic/commands/flashcardcommands/ListFlashcardSetCommand.java

```diff
@@ -21,14 +19,7 @@
     public CommandResult execute(FlashcardModel model) {
         requireNonNull(model);
         model.updateFilteredFlashcardSetList(PREDICATE_SHOW_ALL_FLASHCARDSETS);
-        ObservableList<FlashcardSet> flashcardSets = model.getFlashcardSetList();
 
-        StringBuilder details = new StringBuilder();
-        details.append("\nThere are ");
-        details.append(flashcardSets.size());
-        details.append(" sets");
-        flashcardSets.forEach(flashcardSet -> details.append("\n" + flashcardSet.toString()));
-
-        return new CommandResult(MESSAGE_SUCCESS + details.toString());
+        return new CommandResult(MESSAGE_SUCCESS);
```

Comment:
Do the temporary details disrupt the Command tests? If not, I suggest to keep it until Petrick done with the Flashcard UI.

Reply:
Ah ok! I deleted because the UI should already have listed the sample flashcard sets, but i'll add them back